source/train.py

```diff
@@ -90,13 +139,14 @@ def training_loop(mount_uuid, player, item, mounts, stable, time_elapsing_coeffi
                 cout(COLOR_YELLOW, end="")
                 text_handling.print_long_string(text)
                 cout(COLOR_RESET_ALL, end="")
-        elif choice == 'Train':
+        elif choice.startswith('t'):
```

Comment:
```suggestion
        elif choice.lower().startswith('t'):
```

Reply:
Actually the .lower() function is already applied to the choice variable:
```python
choice = cinput(COLOR_GREEN + COLOR_STYLE_BRIGHT + "> " + COLOR_RESET_ALL).lower()
```